src/aioswitcher/api/__init__.py

```diff
@@ -719,9 +779,21 @@ async def stop_shutter(self) -> SwitcherBaseResponse:
             "logged in session_id=%s, timestamp=%s", login_resp.session_id, timestamp
         )
 
-        packet = packets.RUNNER_STOP_COMMAND.format(
-            login_resp.session_id, timestamp, self._device_id
-        )
+        if self._device_type and self._device_type.token_needed and bool(self._token):
+            command = "0000"
+            hex_pos = f"0{index}{command}" if index else command
```

Comment:
`if index` doesn't make sense since you set above `index+1` so index can't be zero

Reply:
It is done.